### PR TITLE
Simplify LinkPresentationView and make it use its intrinsic size.

### PR DESCRIPTION
**Pull request by Moriquendi**

### CODE REVIEW

**Comment by vmanot** [comment]
@Moriquendi does this work in a SwiftUI `List`? The reason I had to resort to the height hacks was because of issues there.

**Comment by Moriquendi** [comment]
@vmanot Lists looks good to me too. And their size is updated after fetching the link.

Screenshot 2021-08-03 at 18 16 35

Screenshot 2021-08-03 at 18 15 57

File: Sources/Intermodular/Helpers/UIKit/UIHostingCollectionViewCell.swift [hunk 1]
```diff
@@ -271,6 +271,12 @@ class UIHostingCollectionViewCell<
             
             return layoutAttributes
         } else {
+//            let fitSize = systemLayoutSizeFitting(CGSize(width: layoutAttributes.size.width,
+//                                           height: 99999999))
+//
+//            layoutAttributes.size = fitSize
+//            return layoutAttributes
+
```
Comment (vmanot) [inline Sources/Intermodular/Helpers/UIKit/UIHostingCollectionViewCell.swift:279]: Please remove commented out code.

**Review by vmanot — APPROVED** [review]

**Comment by vmanot** [comment]
Closing due to inactivity.

**Pull request closed without merging**